coretex/cli/modules/node.py

```diff
@@ -255,6 +255,23 @@ def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
     return nodeRam
 
 
+def promptSwap(config: Dict[str, Any], swapLimit: int) -> int:
+    nodeSwap: int = clickPrompt(
+        f"Node SWAP memory limit in GB (Maximum: {swapLimit}GB) (press enter to use default)",
+        swapLimit,
+        type = int
+    )
+
+    if nodeSwap > swapLimit:
+        errorEcho(
+            f"ERROR: SWAP memory limit in Docker Desktop ({swapLimit}GB) is lower than the configured value ({nodeSwap}GB). "
+            "Please adjust resource limitations in Docker Desktop settings."
```

Comment:
Is there a need for the 2nd sentence? You can tell user that if they want to use a **higher value** than what is currently configured that they also need to change docker limits, not docker desktop limits.